config/fcos/v1_6_exp/validate.go

```diff
@@ -15,13 +15,26 @@
 package v1_6_exp
 
 import (
+	"regexp"
+
 	"github.com/coreos/butane/config/common"
 	"github.com/coreos/ignition/v2/config/util"
 
 	"github.com/coreos/vcontext/path"
 	"github.com/coreos/vcontext/report"
 )
 
+func (c Config) Validate() (r report.Report) {
+	regex := regexp.MustCompile(`^/(etc|var)/.*`)
+	for _, fs := range c.Storage.Filesystems {
+		fsPath := *fs.Path
+		if len(fsPath) < 4 || !regex.MatchString(fsPath) {
```

Comment:
Hmm, is the len(fsPath) still a necessary check? and I think it might be introducing a bug.

We might want some tests that cover the case of something that is not strictly '/' '/etc' '/var' '/boot' IE, what about a path like '/thisIsABugTest' ?